cmd/trade.go

```diff
@@ -194,12 +199,7 @@ func deleteAllOffersAndExit(botConfig trader.BotConfig, client *horizon.Client,
 	}
 }
 
-func setLogFile() {
-	t := time.Now().String()
-	var b strings.Builder
-	b.WriteString("KelpLog_")
-	b.WriteString(t)
-	fileName := b.String()
+func setLogFile(fileName string) {
 	f, err := os.OpenFile(fileName, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
 	if err != nil {
 		log.Fatalf("Failed to set log file: %v", err)
```

Comment:
this should not log a fatal error here but should wrap the error and return it, example:

    return fmt.Errorf("failed to set log file: %s", e)

the calling code can then check for that error and log a Fatal error like so:

 	if e != nil {
 	 	log.Println()
 	 	log.Fatal(e)
 	 	return
 	 }

also, in Kelp the convention is to use `e` to denote errors instead of using `err`. it's shorter and much easier to read.